applications/blockpublisher.md

```diff
@@ -0,0 +1,151 @@
+# BlockPublisher
+
+- **Team Name:** Reaudito
+- **Payment Details:**
+  - **DOT**: 162QCGd1unek7Guieaxuoekf1oVN3RCg2fbxjwhRVMZ7H4cG
+  - **Payment**: Payment in **USDC or USDT**, Polkadot AssetHub address 162QCGd1unek7Guieaxuoekf1oVN3RCg2fbxjwhRVMZ7H4cG
+
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 2
+
+## Overview
+#### Decentralized Publishing of News, Journals, and Books
+
+The advent of blockchain technology has the potential to transform the publishing industry in several ways. By leveraging decentralized networks and cryptographic techniques, blockchain-based publishing solutions can offer a more efficient, transparent, and accessible system for authors, publishers, and readers alike. It's also versatile enough to be used for entertainment content.
+
+
+
+### Project Details
+
+Key Features of Blockchain-Based Publishing:
+
+- Decentralized Price Discovery: Blockchain-based platforms can facilitate decentralized price discovery, allowing readers to directly support authors by contributing to a community-driven funding pool. This approach ensures that authors receive fair compensation for their work, while readers have greater control over the content they consume.
+- Subsidized Articles through Taxes: By implementing a token-based system, blockchain-based publishing platforms can introduce a novel approach to subsidizing articles. A small portion of the token's value can be allocated to support authors, making high-quality content more accessible to readers.
+- Free Staking Coin for Enhanced Accessibility: To further promote open access and sustainability, blockchain-based publishing platforms can introduce a free staking coin. This coin can be used to purchase articles, remove paywalls, or access premium content, making it more convenient for readers to engage with published works.
+- Open Access and Paywall Removal: The free staking coin can also be used to make articles open access or remove paywalls entirely. This approach can increase readership, promote knowledge sharing, and support the dissemination of high-quality research and content.
+
+## Make the content Open Access Model
+
+### Pasword Sharing:
+Sharing passwords for encrypted files is problematic, in P2P systems. One approach is to store the hash of the password in the blockchain and use Zero Knowledge Proof.
```

Comment:
I think storing the password hash in the blockchain is still less secure than ZKPs as it reveals info about the password's structure.

Reply:
Storing password hashes on blockchain can reveal the structure of a password and may be less secure. However, the purpose of the app is to promote open access and disclose the password after one or two months, once some funding for the article or content has been raised. Therefore, stringent password security is not required. Additionally, users can utilize an auto-password generator to create strong passwords.